pmacApp/src/pmacTrajectory.cpp

```diff
@@ -294,6 +294,25 @@ asynStatus pmacTrajectory::getUserMode(int index, int *user) {
   return status;
 }
 
+asynStatus pmacTrajectory::getVelocityMode(int index, int *velocityMode) {
+  asynStatus status = asynSuccess;
+  static const char *functionName = "readVelocityMode";
```

Comment:
functionName is set to 'readVelocityMode' instead of 'getVelocityMode'

There are a few functions in this section with the same mismatch

Reply:
Done in 00568ed39924bc5af541a8fb5cf7ea871cc899f4